x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -0,0 +1,329 @@
+          pageIndex: tableOptions.pageIndex,
+          sortField: tableOptions.sort.field,
+          sortDirection: tableOptions.sort.direction,
+        },
+      },
+    }).then((response) => {
+      return {
+        items: response.transaction_groups,
+        totalItemCount: response.total_transaction_groups,
+        tableOptions: {
+          pageIndex: tableOptions.pageIndex,
+          sort: {
+            field: tableOptions.sort.field,
+            direction: tableOptions.sort.direction,
+          },
+        },
+      };
+    });
+  }, [
+    serviceName,
+    start,
+    end,
+    uiFilters,
+    tableOptions.pageIndex,
+    tableOptions.sort.field,
+    tableOptions.sort.direction,
+  ]);
+
+  const {
+    items,
+    totalItemCount,
+    tableOptions: { pageIndex, sort },
+  } = data;
+
+  const columns: Array<EuiBasicTableColumn<ServiceTransactionGroupItem>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnName',
+        {
+          defaultMessage: 'Name',
+        }
+      ),
+      render: (_, { name, transaction_type: transactionType }) => {
+        return (
+          <TransactionGroupLinkWrapper>
+            <EuiToolTip delay="long" content={name}>
+              <StyledTransactionDetailLink
+                serviceName={serviceName}
+                transactionName={name}
+                transactionType={transactionType}
+              >
+                {name}
+              </StyledTransactionDetailLink>
+            </EuiToolTip>
+          </TransactionGroupLinkWrapper>
+        );
+      },
+    },
+    {
+      field: 'latency',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency',
+        {
+          defaultMessage: 'Latency',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { latency }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis1"
+            compact
+            series={latency.timeseries ?? undefined}
+            valueLabel={
+              latency.value !== null
+                ? asDuration(latency.value)
+                : NOT_AVAILABLE_LABEL
+            }
+            start={parseFloat(start!)}
+            end={parseFloat(end!)}
+          />
+        );
+      },
+    },
+    {
+      field: 'traffic',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnTraffic',
+        {
+          defaultMessage: 'Traffic',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { traffic }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis0"
+            compact
+            series={traffic.timeseries ?? undefined}
+            valueLabel={
+              traffic.value !== null
+                ? asTransactionRate(traffic.value)
+                : NOT_AVAILABLE_LABEL
+            }
+            start={parseFloat(start!)}
```

Comment:
I don't understand this. Isn't `start` a string like `2020-11-09T19:48:43.778Z`? If that's the case `parseFloat` only returns the year - is that the case?

Reply:
Btw looking closer at this: 
 - We need to pass `start` or `end` to `SparkPlotWithValueLabel` since we don't show any ticks the date range for the empty state doesn't matter
 - Furthermore: the empty state created in `SparkPlotWithValueLabel` seems to be made doubly redundant in `SparkPlot` by the `isEmpty` check

Reply:
No idea how I got here 🤦‍♂️ Will fix.